hilfe/handbuch/umsaetze/tagesabrechnung/index.md

```diff
@@ -107,9 +107,15 @@ Analog zum hier gezeigten Vorgehen können Sie auch mit "Unbaren Zahlungsarten",
 
 Selbst wenn Sie das Kassenbuch von Pentacode nicht nutzen sollten, empfehlen wir, die **Erlöse** in Ihrem Unternehmen zumindest zu **statistischen Zwecken** zu erfassen. Neben einer Darstellung Ihrer Umsätze nach deren Herkunft ermittelt Pentacode automatisch die Arbeits-Produktivität in Ihrem Unternehmen.
 
-Verwenden Sie für die Umsatzeingabe am besten [**Umsatzgruppen**](/hilfe/handbuch/umsaetze/umsatzgruppen/).
+Verwenden Sie für die Umsatzeingabe am besten **Umsatzgruppen**.
+[**Hier**](/hilfe/handbuch/umsaetze/umsatzgruppen/) können Sie nachschlagen, wie
+Sie diese erstellen und ordnen. 
 
-Wenn Sie im Menüpunkt Umsatzgruppen auf "**+ Neue Umsatzgruppe**" klicken, öffnet ein  Eingabefeld, in dem Sie u.a. buchhalterische Angaben leisten können. Wenn Sie Ihre Umsatzarten nur aus statistischen Gründen erfassen, können Sie diese Eingaben ignorieren. **Sie sollten jedoch unbedingt den zutreffenden UST-Satz wählen**.
+Wenn Sie im Menüpunkt Umsatzgruppen auf "**+ Neue Umsatzgruppe**" klicken,
+öffnet sich ein Eingabefeld, in dem Sie u.a. buchhalterische Angaben leisten
+können. Wenn Sie Ihre Umsatzarten nur aus statistischen Gründen erfassen, können
+Sie diese Eingaben ignorieren. **Sie sollten jedoch unbedingt den zutreffenden
+UST-Satz wählen**.
 
 {% figure caption="Ignorieren Sie für die statistische Erfassung die buchhalterischen Angaben, wählen Sie jedoch den zutreffenden UST-Satz" %}
```

Comment:
Hier bin ich mir nicht sicher ob dieser Teil auch in der Tagesabrechnung bestehen soll - der gleiche Inhalt steht schon unter Umsatzgruppen.

Reply:
Also fast alles unter "Statistische Erfassung"